### PR TITLE
[FORMS-9155] fix setFocus using rules

### PR DESCRIPTION
**Pull request by amit-mnnit**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes and the overall coverage did not decrease.
- [ ] All unit tests pass on CircleCi.
- [ ] I ran all tests locally and they pass.

### CODE REVIEW

File: ui.frontend/src/view/FormFieldBase.js [hunk 1]
```diff
@@ -352,6 +352,10 @@ export default class FormFieldBase extends FormField {
             }
         }
     }
+    
+    updateActiveChild(activeChild) {
```
Comment (rismehta) [inline ui.frontend/src/view/FormFieldBase.js:356]: Please write a test case for this use-case

**Review by rismehta — CHANGES_REQUESTED** [review]
Test case missing

File: ui.frontend/src/view/FormFieldBase.js (hunk 1 above)
Comment (deepprakash345) [inline ui.frontend/src/view/FormFieldBase.js:356]: @amit-mnnit , is there any change in af-core side corresponding to this PR??

**Comment by amit-mnnit** [comment]
> Test case missing

@rismehta done. please check

File: ui.frontend/src/view/FormFieldBase.js (hunk 1 above)
Comment (amit-mnnit) [reply ui.frontend/src/view/FormFieldBase.js:356]: done

File: ui.tests/test-module/specs/tabsontop/tabsontop.runtime.spec.js [hunk 2]
```diff
@@ -208,4 +208,37 @@ describe("Form with TabsOnTop Layout Container with Hidden Children", () => {
...
+
+  const pagePath = "content/forms/af/core-components-it/samples/tabsontop/focusWithRule.html";
+  let formContainer = null;
+  beforeEach(() => {
+      cy.previewForm(pagePath).then(p => {
+          formContainer = p;
+      })
+  });
+
+  const tabSelector = 'ol li';
```
Comment (rismehta) [inline ui.tests/test-module/specs/tabsontop/tabsontop.runtime.spec.js:223]: Tab selector should refer BEM class right. Can you change it

Comment (amit-mnnit) [reply ui.tests/test-module/specs/tabsontop/tabsontop.runtime.spec.js]: done

**Review by rismehta — APPROVED** [review]
LGTM

**Pull request merged**